hubitat-mitsubishi-mqtt.groovy

```diff
@@ -313,6 +313,24 @@ void setHeatingSetpoint(BigDecimal setpoint) {
     }
 }
 
+void setRemoteTemperature(BigDecimal temperature) {
+    if (temperature != null) {
+        // 0 tells the code on the HP to switch to the internal sensor
+        BigDecimal remoteTemp = temperature == 0 ? 0 : convertInputToCelsius(temperature)
+        def thermostatOperatingState = device.currentValue('thermostatOperatingState')
+        // Add or subtract 0.5C while operating to actually get it to turn off closer to the desired setpoint
+        if (remoteTemp != 0) {
```

Comment:
This doesn't feel good-- why do we need to change what the sensor is sending back? This logic to me makes more sense in whatever is calling in to set the remote temperature, if we think it's sending a lagging measurement or something. Let's keep the driver acting as a fairly dumb passthrough.